packages/arb-token-bridge-ui/src/util/WithdrawOnlyUtils.ts

```diff
@@ -11,6 +11,15 @@ export type WithdrawOnlyToken = {
 }
 
 const withdrawOnlyTokens: { [chainId: number]: WithdrawOnlyToken[] } = {
```

Comment:
I don't think we need most of the properties beside the L1 address (see L155)

Reply:
- we need the chain to identify where
- we need the symbol for our ease to find out which tokens are listed
- l2 custom address & l2 address are for our reference